### PR TITLE
Add namespace prefix

### PR DESCRIPTION
**Pull request by yito88**
https://scalar-labs.atlassian.net/browse/DLT-7606

Add the specified prefix to each namespace

This PR requires #124 to create tables with the prefix.

### CODE REVIEW

File: src/main/java/com/scalar/db/api/Operation.java [hunk 1]
```diff
@@ -51,6 +79,45 @@ public Operation(Key partitionKey, Key clusteringKey) {
...
+   *
+   * @return an {@code Optional} with the returned the full table name
+   */
+  @Nonnull
+  public Optional<String> forFullTableName() {
+    if (!namespace.isPresent() || !tableName.isPresent()) {
+      return Optional.empty();
+    }
+
+    if (namespacePrefix.isPresent()) {
```
Comment (feeblefakie) [inline src/main/java/com/scalar/db/api/Operation.java:93]: I think it is a bit redundant. How about this?
```    
    StringBuilder builder = new StringBuilder();
    if (namespacePrefix.isPresent()) {
      builder.append(namespacePrefix.get()));
    }
    builder.append(namespace.get());
    builder.append(".");
    builder.append(tableName.get()); 
    return Optional.of(builder.toString());
```

File: src/main/java/com/scalar/db/api/Operation.java [hunk 2]
```diff
@@ -41,6 +53,22 @@ public Operation(Key partitionKey, Key clusteringKey) {
...
   }
 
+  /**
+   * Returns the namespace with the prefix for this operation
+   *
+   * @return an {@code Optional} with the returned namespace with the prefix
+   */
+  @Nonnull
+  public Optional<String> forFullNamespace() {
+    if (namespace.isPresent() && namespacePrefix.isPresent()) {
```
Comment (feeblefakie) [inline src/main/java/com/scalar/db/api/Operation.java:63]: I think it should at least warn if this method is called when `namespace` is not present but `namespacePrefix` is present.
What do you think?

**Review by feeblefakie — COMMENTED** [review]
Overall looking good! Left two suggestions.

File: src/main/java/com/scalar/db/api/Operation.java (hunk 1 above)
Comment (yito88) [reply src/main/java/com/scalar/db/api/Operation.java:103]: Thank you! The suggestion looks better. Modifled.

File: src/main/java/com/scalar/db/api/Operation.java (hunk 2 above)
Comment (yito88) [reply src/main/java/com/scalar/db/api/Operation.java:71]: I've added warning meesages when the `namespace` is not present.

File: src/main/java/com/scalar/db/api/Operation.java [hunk 3]
```diff
@@ -41,6 +57,26 @@ public Operation(Key partitionKey, Key clusteringKey) {
...
 
+  /**
+   * Returns the namespace with the prefix for this operation
+   *
+   * @return an {@code Optional} with the returned namespace with the prefix
+   */
+  @Nonnull
+  public Optional<String> forFullNamespace() {
+    if (!namespace.isPresent()) {
+      LOGGER.warn("namespace isn't specified");
```
Comment (feeblefakie) [inline src/main/java/com/scalar/db/api/Operation.java:68]:
```suggestion
      LOGGER.warn("namespace is not set. So, you might get an unexpected return value.");
```

**Review by feeblefakie — DISMISSED** [review]
LGTM! Thank you!

**Pull request merged**